src/main/java/org/sagebionetworks/bridge/exporter/helper/ExportHelper.java

```diff
@@ -216,4 +251,48 @@ public String uploadFreeformTextAsAttachment(String recordId, String text) throw
         s3Helper.writeBytesToS3(attachmentBucket, attachmentId, text.getBytes(Charsets.UTF_8));
         return attachmentId;
     }
+
+    /**
+     * Helper method to get end date time from sqs request
+     * @param request
+     * @return
+     */
+    public DateTime getEndDateTime(BridgeExporterRequest request) {
+        DateTime endDateTime;
+        if (request.getDate() != null) {
+            endDateTime = request.getDate().toDateTimeAtStartOfDay(timeZone).plusDays(1).minusMillis(1);
```

Comment:
There are some weird edge-cases that come up. It might be cleaner to just say withTime(23, 59, 59, 999) instead.